### PR TITLE
Add appimage package generation

### PR DESCRIPTION
**Pull request by chrisroberts**

### CODE REVIEW

File: package/appimage.sh [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+
+ORIGIN_DIR=$(pwd)
+
+# Get our directory
+SOURCE="${BASH_SOURCE[0]}"
+while [ -h "$SOURCE" ] ; do SOURCE="$(readlink "$SOURCE")"; done
+DIR="$( cd -P "$( dirname "$SOURCE" )" && pwd )"
+
+appimg_dir="${DIR}/appimage"
+gem_path=$(ls "${DIR}"/vagrant*.gem)
```
Comment (briancain) [inline package/appimage.sh]: This currently doesn't exist in the package dir....should it?

Comment (chrisroberts) [reply package/appimage.sh]: no

**Comment by probonopd** [comment]
Can the AppImage be downloaded somewhere?

**Comment by chrisroberts** [comment]
@probonopd Hi! It will be included as a downloadable artifact with the 2.1 release (two releases away). I'm going to be making it available for beta testing after the next release (2.0.4) so we can chase down any edge case issues before it's included in the official release list.

**Pull request merged**